### PR TITLE
Updated documentation of reflection probes

### PR DESCRIPTION
**Pull request by fredericv-unity3d**
### Purpose of this PR
Added documentation for reflection probes

---
### Testing status
Only documentation were added

---

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Reflection probe usage
+
+## What is a probe
+
+Reflection probes and planar reflection probes provide indirect specular lighting.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:5]: [Reflection Probes](Reflection-Probe.md) and [Planar Reflection Probes](Planar-Reflection-Probe.md) provide indirect specular lighting.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+A probe capture its surroundings and will influence specular lighting in a specific volume.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:7]: A probe captures its surroundings and influences specular lighting within a specific volume.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+A reflection probe will capture all of its surroundings while a planar reflection probe will capture only one direction.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:8]: A Reflection Probe captures all of its surroundings whereas a Planar Reflection Probe only captures one direction.

For more information on probes and how they work, see [Reflection Probes](Reflection-Probes-Intro.md).

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Controlling the influence of a probe
+
+The influence of a probe determines which pixels will be affected and by how much.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:12]: The influence of a probe determines which pixels it affects and by how much.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+There are 3 tools to control the influence on a pixel:
+* Influence volume: any pixel inside this volume will be affected.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:15]: * Influence volume: The probe affects any pixel inside this volume.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+* Blend distance: pixel near the border of the volume will be less influenced.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:16]: * Blend distance: The probe affects pixels near the border of the Influence volume less.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+* Blend normal distance: pixels near the border with an invalid normal won't be affected.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:17]: * [Blend normal distance](#blend-normal-influence): The probe does not affect pixels near the border with an invalid normal.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+### Influence volume
+
+Use this volume to include or exclude pixels to be influenced.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:21]: Use this volume to include or exclude pixels from the probe's influence.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+Note: When a pixel is inside an influence volume it will be _processed_ by the probe, even if specular value provided by the probe is not significant. This is important to handle performance of probes.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:23]: Note: When a pixel is inside an influence volume, the probe still _processes_ it even if the specular value provided by the probe is not significant. This is important to handle the performance of probes.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+![img](Images/ReflectionProbe_Influence.svg)
+
+### Blend influence
+
+Between the influence volume and the blend volume, the specular lighting value provided by the probe is linearly weighted.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:29]: Unity weights the specular lighting value provided by the probe linearly between the influence volume and the blend volume.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+Use the blending to make smooth transition at the border of the influences of a probe or when probes are overlapping.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:30]: Use blending to create smooth transitions at the border of the probe's influence, or when probes overlap.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+
+![img](Images/ReflectionProbe_InfluenceBlend.svg)
+![img](Images/ReflectionProbe_InfluenceBlendOverlap.svg)
+
+### Blend normal influence
+
+In some cases, a probe influence can "bleed", it does influence a pixel that makes no sense to be influenced.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:37]: In some cases, a probe can influence a pixel that is not consistent with the environment.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+
+This is the case when a light ray can not reach a pixel due to occlusion but is inside the influence volume.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:39]: For example, when a light ray cannot reach a pixel due to occlusion but is inside the influence volume.

File: com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+
+You can set a blend normal distance similarly to a blend distance.
+Pixels inside the influence volume but outside of the blend normal volume will not be influenced if their normal is pointing outward the probe.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md:42]: The probe does not influence pixels that are inside the influence volume, but outside of the blend normal distance, if their normal points away from the probe.

**Comment by fredericv-unity3d** [comment]
Thanks for the comments they were useful!

Considering image format, I way prefer to keep svg because:
* It is supported in markdown and Github does display it appropriately
* If we need to edit the image, we can just open the SVG file to edit the shapes. Which is not possible for PNG files. (you need to store the source somewhere)

**Comment by sebastienlagarde** [comment]
regarding the comment on svg vs png, can you run test package and doc tool on the PR? 
Just want to be sure that svg is allowed. We have get many problem in the past with anything not being a png (i.e yamato validation or publishing failure). Thanks

**Comment by sebastienlagarde** [comment]
instruction from lewis:

f you install the doc tools package in your project, you can build the docs locally. Let me find the package name and version for you
14:14
    "com.unity.package-manager-doctools": "1.2.1-preview.1",
and you need
  "registry": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates"
after your dependencies in the manifest

hen you just open the package manager and click Generate Documentation on your local package

**Comment by JordanL8** [comment]
SVG is fine in the main manual, just not sure about packages. If it works fine, then keep them in!

**Review by JordanL8 — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**